docs/browsers.md

```diff
@@ -0,0 +1,24 @@
+## Chromeium based (e.g. Chrome) userstyles
+
+To find your browser version, go to `chrome://version/` in your browser.
+
+## Firefox userstyles
+
+To find your browser version, go to `about:support` in your browser.
+
+### Flags
+
+Sometimes in order for userstyles to work properly, you need to enable certain flags. To do this, go to `about:config` in your browser and search for the appropriate flag.
+
+## Opera GX userstyles
+
+To find your browser version, go to `opera://about/` in your browser.
+
+## Opera userstyles
+
+To find your browser version, go to `opera://about/` in your browser.
+
+### Usage
+1. Install the userstyles as you would normally do.
+2. To enable the themes chrome-extension://clngdbkpkpeebahjckkjfobafhncgmne/manage.html and make sure the userstyle is enabled.
+3. To then alter the theme to use your chosen flavor and accent color, click on the gear icon next to the userstyle and select your flavor and accent color.
```

Comment:
This section can be shorter, I don't think it's worth specifying the usage of the flags if we're not doing it for each browser that's listed in the table.

```suggestion
Find your browser version below by typing in the URL for your browser:

<table>
<tr>
 <td> Browser
 <td> URL
<tr>
 <td> Chromium
 <td> <code>chrome://version/</code>
<tr>
 <td> Firefox
 <td> <code>about:support</code>
<tr>
 <td> Opera (GX)
 <td> <code>opera://about/</code>
</table>
```